src/openrct2/world/TileInspector.cpp

```diff
@@ -151,12 +151,14 @@ GameActionResult::Ptr tile_inspector_remove_element_at(const CoordsXY& loc, int1
 {
     if (isExecuting)
     {
-        // Forcefully remove the element
-        TileElement* const tileElement = map_get_nth_element_at(loc, elementIndex);
-        if (tileElement == nullptr)
+        // At least one surface element must exist on a tile
+        if (1 >= windowTileInspectorElementCount)
```

Comment:
We don't use [Yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions) in this project.

```suggestion
        if (windowTileInspectorElementCount < 1)
```